src/Paket.Core/Installation/InstallProcess.fs

```diff
@@ -338,25 +338,13 @@ let private applyBindingRedirects isFirstGroup createNewBindingFiles cleanBindin
         |> Seq.sort
 
     applyBindingRedirectsToFolder isFirstGroup createNewBindingFiles cleanBindingRedirects root allKnownLibNames bindingRedirects
-
-let invalidateRestoreCachesForDotnetSdk (projectFileInfo:FileInfo) =
-    let paketPropsFile = ProjectFile.getPaketPropsFileInfo projectFileInfo
-    if paketPropsFile.Exists then
-        let old = File.ReadAllText paketPropsFile.FullName
-        let newContent = old.Replace("<!-- <RestoreSuccess>False</RestoreSuccess> -->","<RestoreSuccess>False</RestoreSuccess>")
-        File.WriteAllText(paketPropsFile.FullName, newContent)
-    let assetsFile = ProjectFile.getAssetsFileInfo projectFileInfo
-    if assetsFile.Exists then
```

Comment:
@Krzysztof-Cieslak @enricosada  I'm not sure if this will break the ionide integration. FSAuto was watching the assets file, right? does it watch the props files?